spec/models/piece_spec.rb

```diff
@@ -1,5 +1,42 @@
 require 'rails_helper'
 
 RSpec.describe Piece, type: :model do
-  
-end
+  describe "#move_to!" do
+    context "when another piece is on the new spot" do
+      it "captures it by updating its coords to nil" do
+        piece1 = Piece.create(x_cord: 0, y_cord: 0, color: "white")
```

Comment:
Very good test cases!

Please think about the following (as mentioned above with Piece.find_by):

How are piece1 and piece2 related? What if there are two games in the database right now and piece1 is on game1 and piece2 on game2...